src/Activities/Internal/ActivityMetricsSender.cs

```diff
@@ -73,6 +81,16 @@ public void SendActivityMetric(Activity activity)
 				}
 			}
 
+			bool isSetParentNameAsDimensionEnabled = m_activityOptions.Value.SetParentNameAsDimension;
+			if(isSetParentNameAsDimensionEnabled)
+			{
+				Activity? parent = activity.Parent;
+				if (parent != null && !string.IsNullOrEmpty(parent.OperationName))
```

Comment:
```suggestion
				if (!string.IsNullOrEmpty(parent.OperationName))
```I believe the first nullcheck is redundant because it is alreadyt covered by string.isNullOrEmpty